Makefile

```diff
@@ -12,7 +12,7 @@ CIARTIFACTS?=ci-artifacts
 COVERAGEOUT?=coverage.out
 COVERAGEHTML?=coverage.html
 PACKAGENAME?=lacework-cli
-CLINAME?=lacework
+CLINAME?=lacework-dev
```

Comment:
I'm sure this is just for testing purposes, but just calling out so it doesn't get missed later on when opening a PR

Reply:
done